src/death.cpp

```diff
@@ -195,6 +195,13 @@ int32_t playerCalculateTotalPoints() {
     return total;
 }
 
+static uint8_t highScoreGenderLabel() {
```

Comment:
Can we leave this where it was for now. I'd like to keep things to just one type of refactor per PR.